arclight-common/src/main/java/io/izzel/arclight/common/mixin/optimization/general/EntityAccessor.java

```diff
@@ -0,0 +1,14 @@
+package io.izzel.arclight.common.mixin.optimization.general;
+
+import net.minecraft.entity.Entity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import java.util.Set;
+
+@Mixin(Entity.class)
+public interface EntityAccessor {
+
+    @Invoker void callGetRecursivePassengers(boolean playersOnly, Set<Entity> passengers);
```

Comment:
I prefer not introducing new interfaces like this but using existing bridge interfaces directly.

Reply:
Ah yeah, my bad! I've updated them to use the bridges and added methods to said bridges where necessary :)